### PR TITLE
Include raven environment filter in local_settings_template

### LINKED ISSUE #15: Include raven environment filter
We require to include an environment filter for raven logs for recent deployments.  
This allows us to have one Sentry project for all environments (dev and production)

### PR DESCRIPTION
**Pull request by WinnyTroy**
Resolves #15

### CODE REVIEW

File: molecule/default/requirements.yml [hunk 1]
```diff
@@ -23,10 +23,14 @@
   src: https://github.com/onaio/ansible-gpg-import.git
   scm: git
 
+- src: https://github.com/DavisRayM/python.git
+  name: DavisRayM.python
+  scm: git
+
```
Comment (DavisRayM) [inline molecule/default/requirements.yml:29]: I don't think this is needed. The `onaio.django` dependency now describes where to get this package, molecule should install it when it's testing.

_[ansible-django change](https://github.com/onaio/ansible-django/blob/master/meta/main.yml#L13-L24)_

**Comment by ivermac** [comment]
@WinnyTroy First things first, could you please create an issue for this pull request?

File: molecule/default/requirements.yml (hunk 1 above)
Comment (WinnyTroy) [reply molecule/default/requirements.yml:29]: ah..sawa...al remove it

**Comment by WinnyTroy** [comment]
> The commit "Include python dependency" doesn't include the python dependency... Please update the commit message

aaaaahhh...good catch. Let me just squash commits

**Pull request merged**